src/main/java/core/basesyntax/figures/Rectangle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.figures;
+
+public class Rectangle extends Figure {
+    private int length;
+    private int width;
+
+    public Rectangle(String color, String name, int length, int width) {
+        super(color, name);
+        this.length = length;
+        this.width = width;
```

Comment:
The `name` parameter is being passed to the superclass constructor. Ensure that this is necessary, as the superclass might already handle the `name` property. If it's redundant, consider removing it.